### PR TITLE
SISRP-12148 - UX Develop: Implement design revisions to better direct Students about changes to Bearfacts

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.berkeley.edu/browse/SISRP-12148

### CODE REVIEW

**Comment by christianvuerings** [comment]
*Note* Another GL3 requirement that just came in - signed off by Ben & UX people. QA PR is in the making.

**Comment by christianvuerings** [comment]
QA PR at #4622

File: src/assets/templates/widgets/profile/contact.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
   data-cc-spinner-directive="contactInformation.isLoading">
 
   <div data-ng-if="!contactInformation.isErrored">
+    <!-- Bearfacts Message -->
+    <div data-ng-include="'widgets/profile/double_entry.html'" data-ng-if="api.user.profile.roles.student"></div>
```
Comment (johncrossman) [inline src/assets/templates/widgets/profile/contact.html]: My understanding: Double-entry only pertains to preferred name. If that is not true, we need to escalate.

File: src/assets/templates/widgets/profile/double_entry.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="cc-page-widget-profile-note">
+  <i class="fa fa-exclamation-triangle cc-icon cc-icon-gold" aria-hidden="true"></i>
+  <strong>Note:</strong> Updates to your personal information will also need to be
+  <a href="https://bearfacts.berkeley.edu/">completed in Bearfacts</a> during our system transition (Fall 2016).
```
Comment (johncrossman) [inline src/assets/templates/widgets/profile/double_entry.html]: "Bear Facts" instead of " Bearfacts"?

**Comment by johncrossman** [comment]
I believe this PR needs two amendments. Otherwise, it's good to go.

**Comment by christianvuerings** [comment]
@johncrossman amendments made (also QA branch) - checked with Ben and you're correct that we should not show this on the the contact card.

**Comment by johncrossman** [comment]
Thanks, @christianvuerings  Once verified in dev I'll merge qa PR

**Pull request merged**